README.adoc

```diff
@@ -0,0 +1,446 @@
+= python-domino
+
+This library provides bindings for the Domino APIs.  See this documentation for details about the APIs:
+
+* https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/[Latest public Domino APIs]
+* https://dominodatalab.github.io/api-docs/[Legacy APIs]
+
+The latest released version of `python-domino` is `1.0.8`.
+
+== Version compatibility matrix
+
+The `python-domino` library is compatible with different versions of Domino:
+
+[cols=",^"]
+|===
+| Domino Versions | Python-Domino
+
+| 3.6.x or lower
+| http://github.com/dominodatalab/python-domino/archive/0.3.5.zip[0.3.5]
+
+| 4.1.0 or higher
+| https://github.com/dominodatalab/python-domino/archive/1.0.0.zip[1.0.0] or higher
```

Comment:
@ddl-olsonJD Is it possible to provide more granularity here?  Would it be useful to provide links to the full set of`.zip` files?